docs/conf.py

```diff
@@ -20,7 +20,8 @@
 # Uncomment the below if you use native CircuitPython modules such as
 # digitalio, micropython and busio. List the modules you use. Without it, the
 # autodoc module docs will fail to generate with a warning.
-# autodoc_mock_imports = ["digitalio", "busio"]
+autodoc_mock_imports = ["digitalio", "busio", "micropython", "adafruit_register",
```

Comment:
This only needs `["micropython", "adafruit_register"]` since that is all that you import. `struct/ustruct` isn't needed (my mistake earlier; forgot struct is in CPython so Sphinx will import it without issue).

Reply:
Done